packages/extension-link/src/link.ts

```diff
@@ -157,13 +157,23 @@ export const Link = Mark.create<LinkOptions>({
   },
 
   parseHTML() {
-    return [{ tag: 'a[href]:not([href *= "javascript:" i])' }]
+    return [{
+      tag: 'a[href]',
+      getAttrs: dom => {
+        const href = dom.getAttribute('href')
```

Comment:
I think here we need a typecheck as dom apparently can also be a string (at least typescript says so) so you can't get the href directly from a string object as `getAttribute` does not exist